include/albatross/Common

```diff
@@ -34,7 +32,7 @@
 #include <numeric>
 #include <random>
 
-#include <albatross/src/core/declarations.hpp>
+#include "ForwardDeclarations"
```

Comment:
no filetype?

Reply:
Nope, `albatross` uses a similar include pattern to `Eigen`.  Files like `Eigen/Core` include all the headers you'd typically need.  I think this is something that comes out of the use of CRTP which can end up requiring a specific include order, so it's nice to have that explicitly done in one place.